server/planning/planning_download.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+"""Planning Download module"""
+import io
+import logging
+import superdesk
+from werkzeug.utils import secure_filename
+from superdesk.errors import SuperdeskApiError
+
+from flask import send_file
+from superdesk.utc import utcnow
+from .planning_article_export import get_items
+
+
+bp = superdesk.Blueprint('planning_download', __name__)
+logger = logging.getLogger(__name__)
+
+
+@bp.route('/planning_download/events/<_ids>')
+def planning_download_file(_ids):
+    export_service = superdesk.get_resource_service('planning_article_export')
+    items = get_items(_ids.split(','), 'events')
+    exported_text = export_service.export_events_to_text(items)
+    if exported_text:
+        try:
+            ###############
```

Comment:
What's this comment?

Change the variable name `_file`. Variable names should not start with an underscore by conventions (underscore in python denotes a private scope).